pkg/image/validation/kubernetes.go

```diff
@@ -414,3 +415,35 @@ func validateHelmChartDuplicates(charts []image.HelmChart) string {
 
 	return ""
 }
+
+func validateAdditionalArtifacts(ctx *image.Context) []FailedValidation {
+	var failures []FailedValidation
+
+	if _, err := os.Stat(combustion.KubernetesManifestsPath(ctx)); !os.IsNotExist(err) {
+		dirEntries, err := os.ReadDir(combustion.KubernetesManifestsPath(ctx))
+		if err != nil {
+			failures = append(failures, FailedValidation{
+				Error: fmt.Errorf("kubernetes manifests directory could not be read: %w", err),
```

Comment:
You can use this description for the UserMessage and only set Error to the err value. Check its usages for reference.